### PR TITLE
modifications to the user interface in user settings:

### PR DESCRIPTION
**Pull request by AlbertoP57**
- profile settings
- attachments
- avatar
- signature
- notifications
- user block
- password

### CODE REVIEW

File: yafsrc/YetAnotherForum.NET/Controls/EditUsersSuspend.ascx.cs [hunk 1]
```diff
@@ -127,9 +127,17 @@ protected void Page_Load([NotNull] object sender, [NotNull] EventArgs e)
             }
 
             // add items to the dropdown
-            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "DAYS"), "1"));
-            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "HOURS"), "2"));
-            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "MINUTES"), "3"));
+            ListItem Item = new ListItem();
```
Comment (w8tcha) [inline yafsrc/YetAnotherForum.NET/Controls/EditUsersSuspend.ascx.cs]: This will not work and will cause viewstate issues! you can not re-use list items

**Comment by AlbertoP57** [comment]
> Why use uppercase? this looks really ugly. i think it would be better to do this via custom themes!

Well ... it looks ugly to you.  I like uppercase because it conveys attention to the user. Especially in headers when the text below has hexxagerated font size ...

However, while the old versions of the forum were plenty of classes, only the bootstrap classes are present in the new versions. Can we add classes like class="profile-card-title". There are no guidelines ...

Please accept the changes and i will after insert a class.

File: yafsrc/YetAnotherForum.NET/Controls/EditUsersSuspend.ascx.cs (hunk 1 above)
Comment (AlbertoP57) [reply yafsrc/YetAnotherForum.NET/Controls/EditUsersSuspend.ascx.cs]: This works ;) Anyway please accept the changes (or restore that file to the original one).
I suggest to use the original:

```
            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "DAYS"), "1"));
            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "HOURS"), "2"));
            this.SuspendUnit.Items.Add(new ListItem(this.GetText("PROFILE", "MINUTES"), "3"));

```
and add this: 

```
            foreach(ListItem listItem in this.SuspendUnit.Items)
                listItem.Attributes.Add("class", "pr-2");

```

**Comment by w8tcha** [comment]
> Please accept the changes and i will after insert a class.

You can use 

`.page-pagename .card-header   {text-transform: uppercase;} `

**Pull request merged**